musicbot/bot.py

```diff
@@ -293,35 +293,35 @@ def _setup_logging(self):
 
     @staticmethod
     def _check_if_empty(
-        vchannel: discord.abc.GuildChannel, *, excluding_me=True, excluding_deaf=False
+        vchannel: discord.abc.GuildChannel, *, excluding_me=True, excluding_deaf=True
```

Comment:
we should exclude deafened members, they can't hear anything anyways

Reply:
the parameter name is a bit confusing, it basically means "exclude deaf members from being counted as active", so the current behaviour is that we don't play for deafened members
perhaps i should change the name